menas-web/ui/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "menas-web",
+  "version": "2.12.0-AWS-SNAPSHOT",
+  "description": "Menas Client",
+  "main": "index.html",
+  "author": "ABSA Big Data R&D",
+  "license": "Apache License, Version 2.0",
+  "devDependencies": {
+    "@ui5/cli": "^2.8.1"
+  },
+  "config": {
+    "apiUrl": "${API_URL}",
```

Comment:
This should not be here. You should not have to care that you made changes to git each time you want to test deploy. There should be another config file or something.

Reply:
API_URL for a system variable is too broad of a name. Please rename to `MENAS_API_URL`

Reply:
It's being set in the `start_menas_web.sh` script by:
`envsubst < /usr/share/nginx/html/package.json > /usr/share/nginx/html/package-new.json
rm /usr/share/nginx/html/package.json
mv /usr/share/nginx/html/package-new.json /usr/share/nginx/html/package.json`.

The script is run in the Dockerfile.

Reply:
So you need to build `menas_web.war` or docker image per environment?

Reply:
No, the docker image just needs an environment variable `API_URL` which will be filled into package.json. When it comes to the war type deployment, the war can also be automatically modified to substitute the `API_URL`